CaloMC/src/CaloShowerStepFromStepPt_module.cc

```diff
@@ -186,7 +196,7 @@ namespace mu2e {
       procCodes_[13].insert(   {2,12,13,16,17,21,23,29,30,31,34,40,49,58,59} ); // mu-
```

Comment:
They are the number of process codes that we want to throw away. They are enum, but I got lazy and referenced their internal number directly. I'll fix this, even if it will be longer...